pkg/helm/tune_test.go

```diff
@@ -3,13 +3,19 @@ package helm_test
 import (
 	"testing"
 
+	"github.com/blang/semver/v4"
 	"github.com/stretchr/testify/assert"
 	"groundcover.com/pkg/helm"
 	"groundcover.com/pkg/k8s"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
+var (
+	OldKernelSemver = semver.MustParse("5.10.0")
```

Comment:
more meaningful name plz, "MemoryAccountKernelSemver" or something like that

Reply:
Fixed